### ISSUE TITLE
Check if we can introduce "line breaks" in citation strings

### ISSUE THREAD

**Issue by santilland** [eurodatacube/eodash-catalog#98]

**Comment by santilland** [comment]
@FedericoRondoni, you asked if this could be possible. We have been discussing internally, and in principle it would be possible to define some character or html tag that would allow a line break, but the idea is that this string can be easily copied in order to be added anywhere where the data is used.
So if we introduce any kind of special symbol, any other location where the string is pasted won't know how to handle it, and the symbol will appear in their citation.
So in conclusion we think that line breaks should not be expected in the citation string.
I will close the issue, but please let us know if you disagree and we can discuss further.
